LICENSE

```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Sathish Kumar
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
`2022 Immobiliare Labs` no?

Reply:
👍

Reply:
> No this is an error, this LICENSE must not be changed. It should remain unaltered.

Where are we adding our license then? I am not sure about this 🤔

Reply:
In the NOTICE File

Reply:
> It should remain unaltered.

ok, revert it! ;)